### PR TITLE
feat: bump version to 1.0.29 (jade), 5.1.2 (esp-idf)

### PR DESCRIPTION
**Pull request by 1-21gigasats**

### CODE REVIEW

File: device_specific/flash_the_m5stack_core_basic [hunk 1]
```diff
@@ -15,11 +15,11 @@ working_directory="${HOME}/Downloads/diy_jade"
 temp_directory="${working_directory}/temp"
 trap cleanup EXIT
 
-jade_git_tag="1.0.26"
++jade_git_tag="1.0.29"
```
Comment (bitcoin-tools) [inline device_specific/flash_the_m5stack_core_basic:18]: Can you please remove the extra `+` here and on line 18/19 of the other scripts?

Comment (1-21gigasats) [reply device_specific/flash_the_m5stack_core_basic]: sorry, please check again

**Comment by 1-21gigasats** [comment]
I think we need a higher timeout for the ci build

**Comment by bitcoin-tools** [comment]
Planning to test this on the four supported devices.

**Comment by bitcoin-tools** [comment]
Passed testing on macOS.

**Pull request merged**